upgrade/1.2/scripts/upgrade/ncn-upgrade-ceph-nodes.sh

```diff
@@ -187,7 +187,7 @@ if [[ $ssh_keys_done == "0" ]]; then
     ssh_keygen_keyscan "${target_ncn}"
     ssh_keys_done=1
 fi
-ssh $target_ncn -t 'GOSS_BASE=/opt/cray/tests/install/ncn goss -g /opt/cray/tests/install/ncn/suites/ncn-upgrade-tests-storage.yaml --vars=/opt/cray/tests/install/ncn/vars/variables-ncn.yaml validate'
+ssh $target_ncn -t 'GOSS_BASE=/opt/cray/tests/install/ncn goss -g /opt/cray/tests/install/ncn/suites/ncn-upgrade-tests-storage.yaml --vars=/opt/cray/tests/install/ncn/vars/variables-ncn.yaml validate' || true 
```

Comment:
just curious -- why do we need the '|| true'?  are we expecting the tests to fail?